examples/adts.agda

```diff
@@ -31,7 +31,7 @@ record RgbPair : Set where
 -- trivial function with single argument
 
```

Comment:
IMHO it would be nice to:
- add entry to [CHANGELOG](https://github.com/lemastero/agda2scala/blob/main/CHANGELOG.md)
- update version:  0.1.0.3 to 0.1.0.4 in [agda2scala.cabal](https://github.com/lemastero/agda2scala/blob/main/agda2scala.cabal)